### DISCUSSION TITLE
convert data back to vdjca

### DISCUSSION THREAD

**Discussion by eriicdesousa** [milaboratory/mixcr#1500] [Q&A] [answered]
I have a lot of alignment data that was exported to tsv files but this analysis was not done by me so I do not have the vdjca files. My ultimate goal is to convert this data to AIRR format and I think the easiest way would be to convert the tsv files back to vdjca and then to AIRR but I don't find anything in the documentation that says this is possible. Can I convert a tsv with alignments to a vdjca file? Many thanks.

**Comment by mizraelson** [comment, marked as answer]
`.vdjca` is a binary format that holds a lot of information on each alignment and unfortunately it's not possible to convert a `.tsv` file back to `.clns` or `.vdjca`
